soc/bflb/bl70x/Kconfig.defconfig

```diff
@@ -5,9 +5,9 @@
 if SOC_SERIES_BL70X
 
 # On BL702, BL704 and BL706, not all values work, here is a list of some that work:
-# 2000, 1000, 100, 10, 1
+# 1000, 100, 10, 1
 config SYS_CLOCK_TICKS_PER_SEC
-	default 2000
+	default 1000
```

Comment:
In both case, it seems like setting the `SYS_CLOCK_TICS_PER_SEC` above 1000 ticks per second is causing troubles...

```
  .  time unit
  |  sys tick irq

2000 ticks/sec ....|....|....|....|....|....|....
1000 ticks/sec .........|.........|.........|....
                   ^
                  one possible difference is the absence of interrupt
                  early-on in the boot process
```

Is the systick modification also helping with getting things to work?

Reply:
That is possible.